frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/ActionUtils.java

```diff
@@ -5,6 +5,10 @@
 public final class ActionUtils {
   private ActionUtils() {}
 
+  public static String createRefspec(String sourceBranchFullName, String targetBranchFullName, boolean allowNonFastForward) {
+    return (allowNonFastForward ? "+" : "") + sourceBranchFullName + ":" + targetBranchFullName;
```

Comment:
So I assume that the current version is so descriptive that we can consciously drop the comment
`// Note the '+' sign preceding the refspec. It permits non-fast-forward updates.`
right?

Reply:
Yes, that was exactly the idea